docklets/Battery/BatteryPreferences.vala

```diff
@@ -0,0 +1,31 @@
+//
```

Comment:
This file is not needed, and can be dropped.

Reply:
My intention was to leave these files in case someone wanted to expand on my work later, nonetheless, I'll sort this out when I have time if you really want me to remove it.